runtime/taskcontext.go

```diff
@@ -195,6 +212,15 @@ func (c *TaskContext) NewLogReader() (io.ReadCloser, error) {
 	return c.logStream.NextReader()
 }
 
+func (c *TaskContext) ExtractLog() (ioext.ReadSeekCloser, error) {
+	file, err := os.Open(c.logLocation)
```

Comment:
@walac Do you have a moment to implement the error handling Jonas suggested above? It's a great idea to prevent races here.